src/TodoList.js

```diff
@@ -9,8 +10,8 @@ const todoList = [
 function TodoList() {
   return (
     <ul>
-      {todoList.map((item) => (
-        <li key={item.id}>{item.title}</li>
+      {todoList.map((props) => (
+        <TodoListItem id={props.id} title={props.title} />
```

Comment:
Requires changes:
When iterating over an array of items, React needs a `key` property to distinguish which item in the list belongs to which item in the DOM: https://react.dev/learn/rendering-lists

As mentioned in the link above, not providing a key with throw an error in the console:
> Warning: Each child in a list should have a unique “key” prop.